tests/test_python_cli.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Can we move this under `python/` as this only tests the python package?

Reply:
I'd rather not - I don't think we gain anything by moving it, and its probably CI-run debugging to check that we copy/reference it correctly from a different path. Its part of CCF testing the package (and in the same style as `test_install.sh`, containing CCF docs annotations), rather than part of the package itself. When/if we move to click, we should probably replace this with a test folder/app/whatever they recommend in the package itself, but I think this is fine for now.